### DISCUSSION TITLE
Donde se subirán las practicas de la frontend mission 01 -Intro

### DISCUSSION THREAD

**Discussion by JonathanGrimaldo** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#122] [Q&A] [answered]
Hola 👋🏻  acabo de concluir las actividades de la [Front-Mission-Intro-01](https://github.com/LaunchX-InnovaccionVirtual/FrontEnd-Mision/tree/main/01%20-%20INTRO) ✏️  mmm pero tengo la duda de donde subiremos las actividades, 🤔  puesto que mañana 25 es la fecha limite, si alguien sabe por favor deje su comentario. 💬🤟🏻

**Comment by villegas97** [comment]
Van a crear un microsoft form, aún no proporcionan el link, seguramente lo pasaran por medio de teams.

**Reply by JonathanGrimaldo** [reply]
De acuerdo, muchas gracias @villegas97 🚀 👍🏻

**Comment by romarpla** [comment, marked as answer]
[Link de publicación de Teams](https://teams.microsoft.com/l/message/19:8a61b3259c3147259ba0197e9b103432@thread.tacv2/1645814720628?tenantId=4ae54b05-b77e-4224-aef1-8661422e0816&groupId=d38ceaac-3599-4f30-94ba-740ccb530cb0&parentMessageId=1645814720628&teamName=Misi%C3%B3n%20Frontend%20-%20JS&channelName=Semana%201%20-%20Despegue%20%F0%9F%9A%80&createdTime=1645814720628)

Este es el link dentro del equipo de Teams para que puedan ver el link del Formulario
